RecoEgamma/EgammaIsolationAlgos/plugins/ParticleBasedIsoProducer.cc

```diff
@@ -184,3 +188,27 @@ void ParticleBasedIsoProducer::produce(edm::Event& theEvent, const edm::EventSet
   fillerElectrons.fill();
   theEvent.put(std::move(eleToPFCandIsoMap_p), valueMapElePFCandIso_);
 }
+
+void ParticleBasedIsoProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  // particleBasedIsolation
+  edm::ParameterSetDescription desc;
+  desc.add<std::string>("valueMapEleToEG", "");
+  desc.add<std::string>("valueMapPhoToEG", "valMapPFEgammaCandToPhoton");
+  desc.add<edm::InputTag>("electronTmpProducer", edm::InputTag("gedGsfElectronsTmp"));
+  desc.add<edm::InputTag>("pfCandidates", edm::InputTag("particleFlow"));
+  desc.add<std::string>("valueMapElePFblockIso", "gedGsfElectrons");
+  desc.add<edm::InputTag>("electronProducer", edm::InputTag("gedGsfElectrons"));
+  desc.add<edm::InputTag>("photonTmpProducer", edm::InputTag("gedPhotonsTmp"));
+  desc.add<edm::InputTag>("pfEgammaCandidates", edm::InputTag("particleFlowEGamma"));
+  {
+    edm::ParameterSetDescription psd0;
+    psd0.add<std::string>("ComponentName", "pfBlockBasedIsolation");
+    psd0.add<double>("coneSize", 9999999999);
```

Comment:
could something like `std::numeric_limits<double>::max()` be better here?

Reply:
Good idea!

Reply:
...but it didn't work. I'll revert this change